evildmp/vanilla-sphinx-test
# Ensure anchor link symbol is only visible on hover

### Done

- Add styles to ensure anchor link symbol is only visible on hover. 
- The link icon needs styling. There is an [issue](https://github.com/canonical-web-and-design/vanilla-framework/issues/3863) for this on Vanilla, not sure when we will be able to schedule it in. 

### QA
- Git checkout this branch
- Run `make run` (or follow the [readme](https://github.com/evildmp/vanilla-sphinx-test#readme) steps if it's the first time you're running the project)
- Go to http://0.0.0.0:8001/about-repository/repertoire/
- Check the anchor links are only visible when you hover over the heading

### Fixes

#12
